client/src/components/SudoModePasswordField/SudoModePasswordField.scss

```diff
@@ -0,0 +1,55 @@
+// entwine component before the react component has loaded
+// styles are set to prevent a FOUT
+.SudoModePasswordField {
+  min-height: 108px;
+
+  @include media-breakpoint-up(lg) {
+    min-height: 140px;
+  }
+
+  .form__field-holder input {
+    display: none;
+  }
+}
+
+// React component
+.sudo-mode-password-field {
+  @include media-breakpoint-up(lg) {
+    width: 100%;
+    max-width: 700px;
+    margin-left: $form-check-input-gutter;
+  }
+
+  &__inner {
+    margin-bottom: 0;
+    padding-bottom: 1rem;
+  }
+
+  &__notice {
+    margin-bottom: 0;
+  }
+
+  &__notice-button {
+    margin-right: 1rem;
+  }
+
+  &__notice-help {
+    margin-left: 3px;
+  }
+
+  &__verify {
+    margin-top: 1rem;
+  }
+
+  &__verify-form-group.form-group {
```

Comment:
Why does this need to specify both class names? Is `sudo-mode-password-field_verify-form-group` ever used without `form-group`? And are there ever any other `form-group` inside `sudo-mode-password-field`?

If no to either of those, update to just use the one class name here in the selector. No need to be more specific than is required, that just makes overriding styles harder.

Reply:
It needs to be this specific to override `.cms-edit-form .form-group`